src/modules/screenings/tests/schema.spec.ts

```diff
@@ -1,34 +1,41 @@
 import { omit } from 'lodash/fp'
 import { parse, parseInsertable } from '../schema'
+import { fakeScreeningFull } from './utis'
```

Comment:
typo in ./utis import and utis.ts filename

Reply:
Thanks! just corrected it